cfme/tests/control/test_actions.py

```diff
@@ -352,7 +262,7 @@ def test_action_suspend_virtual_machine_after_starting(request, vm, vm_off, poli
     policy_for_testing.assign_actions_to_event("VM Power On", ["Suspend Virtual Machine"])
     request.addfinalizer(policy_for_testing.assign_events)
     # Start the VM
```

Comment:
I would changes this inline comment. as vm started and we are just ensuring status.